contracts/contracts/harvest/Dripper.sol

```diff
@@ -49,7 +49,7 @@ contract Dripper is Governable {
 
     struct Drip {
         uint64 lastCollect; // overflows 262 billion years after the sun dies
-        uint192 perBlock; // drip rate per block
+        uint192 perSecond; // drip rate per block
```

Comment:
nit the comment can also be updated to `per second`

Reply:
Thanks, have fixed the comment